applications/TrilinosApplication/custom_processes/trilinos_levelset_convection_process.h

```diff
@@ -342,6 +316,65 @@ class TrilinosLevelSetConvectionProcess
     ///@name Private Operations
     ///@{
 
+    void InitializeConvectionStrategy(BuilderSolverPointerType pBuilderAndSolver)
+    {
+        KRATOS_TRY
+
+        // Get auxiliary member variables from base class
+        auto& r_base_model_part = BaseType::mrBaseModelPart;
+        const auto& r_level_set_var = *BaseType::mpLevelSetVar;
+        const auto& r_convect_var = *BaseType::mpConvectVar;
+
+        // Check the nodal database of the current partition
+        int n_nodes = r_base_model_part.NumberOfNodes();
+        int n_elems = r_base_model_part.NumberOfElements();
+
+        if (n_nodes > 0){
```

Comment:
I am pretty sure this check is done in the var utils, I would skip it

Reply:
It is not. https://github.com/KratosMultiphysics/Kratos/blob/c795fb57657698453623bdaf98549b30e7d2af72/kratos/utilities/variable_utils.h#L965-L979

Reply:
> It is not.
> 
> https://github.com/KratosMultiphysics/Kratos/blob/c795fb57657698453623bdaf98549b30e7d2af72/kratos/utilities/variable_utils.h#L965-L979

???
it is :D
Will no nothing if `rNodes` is empty

Reply:
Technically the check isn't there. Anyway, removing since the empty model part is checked below.

Reply:
yes why would you add it?
I am confused, why would looping an empty container be an issue?